spec/jobs/handover_follow_up_job_spec.rb

```diff
@@ -0,0 +1,127 @@
+require 'rails_helper'
+
+RSpec.describe HandoverFollowUpJob, type: :job do
+  include ActiveJob::TestHelper
+
+  let(:pom) do
+    OpenStruct.new  staff_id: 99999,
+                    first_name: "MARY",
+                    last_name: "JAMES",
+                    agency_id: "LEI",
+                    full_name: "JAMES, MARY",
+                    email_address: "mary.james@someprison.gov.uk"
+  end
+
+  let(:indeterminate_without_ted) do
+    build_offender(sentence_type: :indeterminate, release_date: nil, ted: nil)
+  end
+
+  let(:determinate_offender) do
+    build_offender(sentence_type: :determinate, release_date: Time.zone.today + 8.months, ted: nil)
+  end
+
+  it "does not queue jobs for LDU's without email addresses" do
```

Comment:
IMHO this pattern doesn't really prove enough - it shoulld be showing that the email gets sent with the correct parameters. There is an example in responsibilities_controller_spec.rb